### PR TITLE
add v4.1.0 release notes

### PR DESCRIPTION
**Pull request by KanjiMonster**
Generated with

  ./changelog.sh -v v4.0.0 v4.1.0

Signed-off-by: Jonas Gorski <jonas.gorski@bisdn.de>

### CODE REVIEW

File: release-notes.txt [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+! New in this release
+- The port LEDs on platform Celestica Questone 2A are now functional.
+
+! Changed behaviour
+- Due to FRR 8.0.1 now enforcing link IPv6 local addresses for various
+  IPv6 based protocols, IPv6 Link local addresses are now enabled by
+  default on all interfaces.
+- SSH over IPv6 will now also be blocked by default on non management
+  ports.
```
Comment (ideaship) [inline release-notes.txt:13]: This phrasing does not make it clear what "also" refers to. SSH? IPv6? Non-managment ports? Besides, this sounds like users need to configure extra table entries if they want to forward ssh traffic.

File: release-notes.txt [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+! Bug fixes
+- Fixed a longstanding issue where the switch stopped forwarding
+  multicast and broadcast packets, breaking protocols like DHCP.
+- Fixed a socket leak in baseboxd, which could lead to various issues
+  after several days of uptime.
+- Fixed various issues in multicast handling in baseboxd.
```
Comment (ideaship) [inline release-notes.txt:20]: Drop at least one "various" in lines 18 to 20.

**Review by ideaship — CHANGES_REQUESTED** [review]

File: release-notes.txt (hunk 1 above)
Comment (KanjiMonster) [reply release-notes.txt:13]: How about

> SSH connections to the switch on non management ports will now also be blocked for IPv6.

Comment (ideaship) [reply release-notes.txt:13]: Perfect.

File: release-notes.txt (hunk 2 above)
Comment (KanjiMonster) [reply release-notes.txt]: replaced various issues with multiple issues.

**Pull request merged**